modules/appeals_api/config/schemas/v2/10182_headers.json

```diff
@@ -12,6 +12,7 @@
     "nodCreateParameters": {
       "type": "object",
       "properties": {
+        "X-VA-SSN":             { "$ref": "#/definitions/X-VA-SSN" },
```

Comment:
I added SSN as a header to NOD to get index action working. It wouldn't be required outside of that index action, but would throw an error requiring needing the SSN on the index. Is this problematic anyway?